go.mod

```diff
@@ -1,18 +1,28 @@
 module github.com/jenkins-x-apps/jx-app-jacoco
 
 require (
-	github.com/hashicorp/go-retryablehttp v0.5.0
-	github.com/jenkins-x/ext-jacoco v0.0.82
-	github.com/jenkins-x/jx v1.3.445
-	github.com/pkg/errors v0.8.0
-	k8s.io/api v0.0.0-20180628040859-072894a440bd
-	k8s.io/apiextensions-apiserver v0.0.0-20180621085152-bbc52469f98b
-	k8s.io/apimachinery v0.0.0-20180621070125-103fd098999d
-	k8s.io/client-go v8.0.0+incompatible
-	k8s.io/code-generator v0.0.0-20180621065459-6702109cc68e
-	k8s.io/gengo v0.0.0-20180718083919-906d99f89cd6
-	k8s.io/helm v2.7.2+incompatible
-	k8s.io/kube-openapi v0.0.0-20180719232738-d8ea2fe547a4
-	k8s.io/metrics v0.0.0-20180620010437-b11cf31b380b
-	k8s.io/test-infra v0.0.0-20181016234544-2c26f647f17a
+	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
+	github.com/hashicorp/go-retryablehttp v0.5.2
+	github.com/jenkins-x/jx v1.3.844
```

Comment:
It sucks a bit to depend on the whole of jx. It would be nice to just share some common utilities. Upgrading to the latest jx version was a nightmare.

Reply:
It would be nice to refactor jx to allow easier reuse of the common functionality.

Reply:
Well the Go compiler will only actually pull in the parts it needs, right?

But yes dealing with `go.mod` is a chore.